docs/sql-ref-syntax-ddl-create-table-like.md

```diff
@@ -57,6 +57,8 @@ CREATE TABLE [IF NOT EXISTS] table_identifier LIKE source_table_identifier
 * **TBLPROPERTIES**
 
     Table properties that have to be set are specified, such as `created.by.user`, `owner`, etc.
+    Note that a basic set of table properties defined in a source table is copied into a new table if the format is the same with each other.
```

Comment:
Can we be more specified here? Like `... if the table format including data source provider and storage input format are the same...`

Reply:
of course